AzureDevOpsPolicyConfigurator/Argument/Arguments.cs

```diff
@@ -0,0 +1,147 @@
+﻿using System;
+
+namespace AzureDevOpsPolicyConfigurator.Argument
+{
+    /// <summary>
+    /// Arguments class.
+    /// </summary>
+    internal class Arguments
```

Comment:
We can use an existing command line parser (like [spectre.cli](https://github.com/spectresystems/spectre.cli), [blog post](https://www.patriksvensson.se/2018/04/an-introduction-to-spectre-cli)) instead of writing this ourselves